index.html

```diff
@@ -1,19 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-  <meta charset="utf-8">
-  <title> Athaliah's Portflio </title>
-  <link rel="stylesheet" href="style.css">
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap"
-    rel="stylesheet">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Athaliah's Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css" integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+  <!-- <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css"> -->
+
+  <link rel="stylesheet" href="./style.css" />
+		<link rel="stylesheet" href="./pop.css" />
 </head>
-
 <body>
-
+  <header class="navbar">
```

Comment:
- [ ] When I start your project, I realize that the header section which holds your menu seems to not be properly styled and this makes it wrap the currently looking awesome design.

**This is how your desktop design looks like:**

[Screencast from 24-11-2022 12:09:33 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/203743066-efa5274d-9769-4932-8053-02f76146aa2a.webm)

**This is how your mobile version design looks like:**

[Screencast from 24-11-2022 12:10:19 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/203743211-1f3ce8f9-fafc-4cfb-92e8-1773d56d8a80.webm)

**Kindly look into it and resolve this**